active/website.md

```diff
@@ -0,0 +1,63 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Chair, Co-chair and Board Liaison can make final decision of  sections of the website that might not conform with DSF's CoC.
```

Comment:
```suggestion
```

If there were CoC-related problems on the site, the final decision would be for the CoC committee / working group.